specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2015-03-01-preview/applications.json

```diff
@@ -425,6 +371,27 @@
             "type": "string",
             "description": "The subscription credentials which uniquely identify Microsoft Azure subscription. The subscription ID forms part of the URI for every service call."
         },
+        "ResourceGroupNameParameter": {
+            "name": "resourceGroupName",

```

Comment:
this parameter should be specified on the method and not on the client - i.e. please add `"x-ms-parameter-location": "method"` property.

Reply:
Fixed.